ohttp/Cargo.toml

```diff
@@ -23,12 +25,21 @@ byteorder = "1.3"
 chacha20poly1305 = {version = "0.8", optional = true}
 hex = "0.4"
 hkdf = {version = "0.11", optional = true}
-hpke = {version = "0.10.0", optional = true, default-features = false, features = ["std"]}
 lazy_static = "1.4"
 log = {version = "0.4.0", default-features = false}
 rand = {version = "0.8", optional = true}
 sha2 = {version = "0.9", optional = true}
 thiserror = "1"
+hpke = {version = "0.10.0", optional = true, default-features = false, features = ["std", "x25519"]}
+
+[dependencies.hpke-pq]
```

Comment:
I haven't tried this, but you might be able to do this:

```toml
[dependencies.'cfg(all(feature = "pq", feature = "rust-hpke"))'.hpke]
package = "hpke"
version = "0.10.0"
# plus the features stuff, but optional = false in this case

[dependencies.'cfg(all(not(feature = "pq"), feature = "rust-hpke"))'.hpke]
git = "https//..."
rev = "..."
# plus the features stuff, but optional = false in this case
```

Then you would avoid the conditional `use X as hpke` code.

Reply:
I tried this and it's not supported.